tracer/contrib/gorilla/muxtrace/muxtrace.go

```diff
@@ -56,28 +56,24 @@ func (m *MuxTracer) HandleFunc(router *mux.Router, pattern string, handler http.
 
 // span will create a span for the given request.
 func (m *MuxTracer) trace(req *http.Request) (*http.Request, *tracer.Span) {
-	resource := getResource(req)
+	route := mux.CurrentRoute(req)
+	path, err := route.GetPathTemplate()
+	if err != nil {
+		path = "unknown" // FIXME[matt] when will this happen?
```

Comment:
It will happen if the route does not define a path: https://github.com/gorilla/mux/blob/master/route.go#L550
Maybe we can convert this `FIXME` to a comment stating that.